### ISSUE TITLE
free durries for everybody

### ISSUE THREAD

**Issue by ghost** [wiiam/IrcBot#6]
[19:43:52]  +taylorswift:   You currently have $6000 in the bnz
[19:43:54]  &Sander:    wait
[19:43:56]  &Sander:    .durry
[19:43:56] -taylorswift-    uve bought a durry for $10
[19:43:56]  +taylorswift:                  )
[19:43:56]  +taylorswift:                 (
[19:43:56]  +taylorswift:    _ ___________ )
[19:43:56]  +taylorswift:   [_[___________#
[19:43:58]  &Sander:    .money
[19:43:59]  +taylorswift:   You currently have $6000 in the bnz

**Comment by ineeee** [comment]
can confirm

**Comment by r3c0d3x** [comment]
I don't see the problem here.

**EDIT**: Jokes aside, here's the explanation:

There's something different between the `durry` command's code and any other command's code. If you take a look at [the code handling the `durry` command](https://github.com/wiiaam/taylorswift/blob/8bcab7e8027d7abd343f0f818e727bd956572786/src/main/scala/modules/Money.scala#L220-L239), you will see that there are no calls to the [`write` method](https://github.com/wiiaam/taylorswift/blob/8bcab7e8027d7abd343f0f818e727bd956572786/src/main/scala/modules/Money.scala#L281-L288), which handles re-writing balances. If you take a look at [the code handling the `give` method](https://github.com/wiiaam/taylorswift/blob/8bcab7e8027d7abd343f0f818e727bd956572786/src/main/scala/modules/Money.scala#L242-L277) below it though, you will see that [it uses that method](https://github.com/wiiaam/taylorswift/blob/8bcab7e8027d7abd343f0f818e727bd956572786/src/main/scala/modules/Money.scala#L273-L274).

Basically, whoever wrote the `durry` command's handling code forgot to subtract da monies.

**Comment by wiiaam** [comment]
i fixed it

**Comment by ineeee** [comment]
why
